--- paso/src/orig.SystemMatrix_copyRemoteCoupleBlock.cpp	2021-08-30 19:17:14.000000000 +1000
+++ paso/src/SystemMatrix_copyRemoteCoupleBlock.cpp	2021-08-30 19:17:33.000000000 +1000
@@ -265,7 +265,7 @@
     // allocate pattern and sparse matrix for remote_coupleBlock
     Pattern_ptr pattern(new Pattern(row_coupleBlock->pattern->type,
                         overlapped_n, num_couple_cols, ptr_ptr, ptr_idx));
-    remote_coupleBlock.reset(new SparseMatrix(row_coupleBlock->type,
+    remote_coupleBlock.reset(new SparseMatrix<double>(row_coupleBlock->type,
                              pattern, row_block_size, col_block_size, false));
 
     // send/receive value array
